### ISSUE TITLE
stream.destroy not working

### ISSUE THREAD

**Issue by sudhakar3697** [paulmillr/readdirp#103]
I want to check if a directory has at least a sub-directory.

```
const stream=readdirp(dir, {depth:1,type:'directories', alwaysStat: false})
  .on('data', (entry) => {
    if(entry.dirent.isDirectory()){
    console.log(dir+" has SubDir");
   stream.destroy();
    }
  })
```

Thank you.

**Comment by paulmillr** [comment]
It’s destroy(), not destoy().

**Comment by sudhakar3697** [comment]
@paulmillr  Sorry for the typo. I tried with destroy only.

**Comment by paulmillr** [comment]
should have been fixed with 3.2
